packages/tailwind/package.json

```diff
@@ -19,12 +20,20 @@
     "src",
     "dist"
   ],
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.cjs",
+  "module": "dist/esm/index.mjs",
+  "types": "dist/types/index.d.ts",
   "author": "",
   "private": false,
   "license": "MIT",
-  "dependencies": {
-    "@kaizen/design-tokens": "workspace:*"
+  "devDependencies": {
+    "@kaizen/components": "workspace:*",
+    "@kaizen/design-tokens": "workspace:*",
```

Comment:
I think design-tokens is still a dep for TW right? Or a peerDep if you'd prefer.